README.md

```diff
@@ -15,25 +15,30 @@ A general purpose HTTP client built with extensibility in mind.
 * promise based - no callbacks
 
 ## Contents
-* [Installation](#installation)
-* [Usage](#usage)
-  * [More Examples](#more-examples)
-* [API](#api)
-  * [Global Defaults](#global-defaults)
-  * [`ServiceClient.create(servicename, [overrides])`](#serviceclientcreateservicename-overrides)
-  * [`ServiceClient.read(response, [options])`](#serviceclientreadresponse-options)
-  * [`ServiceClient.mergeConfig({})`](#serviceclientmergeconfig)
-  * [`ServiceClient.use([])`](#serviceclientuse)
-  * [ServiceClient instance](#serviceclient-instance)
-* [Plugins](#plugins)
-  * [Overview](#overview)
-  * [Registration](#registration)
-  * [Available Plugins](#available-plugins)
-  * [Plugin structure](#plugin-structure)
-  * [Writing your own plugin](#writing-your-own-plugin)
-* [Precautionary Notes](#precautionary-notes)
-* [Development](#development)
-* [Further Reading](#further-reading)
+- [@vrbo/service-client](#vrboservice-client)
```

Comment:
Please rollback this change. There is no need to be so specific and provide every header in the Contents, i.e. we can ignore headers which go before the Contents section itself and there is no need to include 3+ level headers as it reduces readability.